meinberlin/apps/contrib/templates/meinberlin_contrib/component_library.html

```diff
@@ -422,6 +434,68 @@ <h2 id="cl-complete">Complete</h2>
                     </div>
                 </div>
 
+                <h2 id="cl-component-library">Component library</h2>
+
+                <div class="cl-example">
+                    <div class="cl-example">
+                        Every component in the component library has a border and some padding.
+                    </div>
+                </div>
+
+                <h2 id="cl-copyright">Copyright</h2>
+
+                <div class="cl-example">
+                    <div class="copyright">
+                        The copyright class is used wherever an image has a copyright.
+                    </div>
+                </div>
+
+                <h2 id="cl-dashboard-nav">Dashboard Nav</h2>
+
+                <div class="cl-example">
+                    <nav class="dashboard-nav">
+                        <div class="dashboard-nav__dropdown">
+                            <div class="dashboard-nav__item">Dashboard nav item (in a dashboard-nav__dropwown, which is not used as dropdown)</div>
+                        </div>
+                        <ul class="dashboard-nav__pages">
+                            <li class="dashboard-nav__page">
+                                <a href="#"
+                                    class="dashboard-nav__item dashboard-nav__item--interactive">
+                                    Interactive Dashboard nav item
+                                </a>
+                            </li>
+                            <li class="dashboard-nav__page">
+                                <a href="#"
+                                    class="dashboard-nav__item dashboard-nav__item--interactive">
+                                    These interactive items are shown in a list
```

Comment:
maybe we should also add the error case with the red exclamation mark icon, as it is specifically layouted to be on the right side of the menu item